admin/operational.js

```diff
@@ -71,6 +72,32 @@ function testConnection () {
     });
 }
 
+function setActualTime () {
+    utils.getDeviceId(function (id) {
+        var cloud = Cloud.init(logger, id);
+        cloud.getActualTime(function (time) {
+            var r = 0;
```

Comment:
Using process.exit only once, after if-else statement was not working as it should. 
It was exiting main process before exec command(it is asynchronous) finished it work and in fact message from system was lost. (since parent process ends before child)
I can yet try with synchronous version of 'exec' but it is available since node.js v.0.12 so an update might be needed.
Is there better solution to that?